packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -87,6 +87,11 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   description?: string;
 
+  /**
+   * Optional custom renderer for the description
+   */
+  onRenderDescription?: IRenderFunction<ITextFieldProps>;
```

Comment:
>onRenderDescription [](start = 2, length = 19)

Please add this along with the rest of the onRender APIs below

Reply:
Added periods to the descriptions of all onRender.... functions